### PR TITLE
CodeCleanUp - Address issues #2096, #2100, #1924

### LINKED ISSUE #2096: Reuse `validate()` logic across `TokenMint`/`TokenBurn`/`TokenWipe` transitions
### Problem

The `validate()` implementations in the `TokenMint`/`TokenBurn`/`TokenWipe` transitions are quite similar.

### Solution

Extract common logic into a helper method, call from all three transitions.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by anighanta**
**Description**:

**Related issue(s)**:

Fixes #2096, #2100, #1924 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/txns/token/TokenOpsValidator.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+					if ((long) serialNum <= 0) {
+						return INVALID_NFT_ID;
+					}
+				}
+			}
+		}
+		return OK;
+	}
+
+	private TokenOpsValidator()  {
```
Comment (tinker-michaelj) [inline hedera-node/src/main/java/com/hedera/services/txns/token/TokenOpsValidator.java:103]: You can cover this constructor by adding `TokenOpsValidator.class` [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/test/java/com/hedera/services/utils/UtilsConstructorTest.java#L66).

**Review by tinker-michaelj — DISMISSED** [review]
LGTM!

File: hedera-node/src/main/java/com/hedera/services/txns/token/TokenOpsValidator.java (hunk 1 above)
Comment (qnswirlds) [reply hedera-node/src/main/java/com/hedera/services/txns/token/TokenOpsValidator.java:103]: Exactly :D. That's the common place to test private constructor for a utility class.

File: hedera-node/src/test/java/com/hedera/services/txns/token/TokenOpsValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+@ExtendWith(MockitoExtension.class)
+class TokenOpsValidatorTest {
+
+	@Mock
+	private OptionValidator validator;
+
+	@BeforeEach
+	private void setup() {
+
+	}
```
Comment (ljianghedera) [inline hedera-node/src/test/java/com/hedera/services/txns/token/TokenOpsValidatorTest.java:58]: nit: if this method doesn't do anything, you don't need it to be here.

**Review by ljianghedera — DISMISSED** [review]
LGTM

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Review by abhishek-hedera — APPROVED** [review]
LGTM

**Review by qnswirlds — APPROVED** [review]

**Pull request merged**